### PR TITLE
texture lod

### PR DESCRIPTION
**Pull request by shrekshao**
Without lod_bias for texobj

### CODE REVIEW

File: samples/texture_lod.html [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+            gl.texParameterf(gl.TEXTURE_2D, gl.TEXTURE_MIN_LOD, 0.0);
+            gl.texParameterf(gl.TEXTURE_2D, gl.TEXTURE_MAX_LOD, 0.0);
+            gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA, gl.RGBA, gl.UNSIGNED_BYTE, image);
+            gl.generateMipmap(gl.TEXTURE_2D);
+            
+            textures[Corners.TOP_RIGHT] = gl.createTexture();
+            gl.bindTexture(gl.TEXTURE_2D, textures[Corners.TOP_RIGHT]);
+            gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MIN_FILTER, gl.LINEAR_MIPMAP_LINEAR);
+            gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MAG_FILTER, gl.LINEAR);
+            gl.texParameterf(gl.TEXTURE_2D, gl.TEXTURE_MIN_LOD, 3.0);
```
Comment (pjcozzi) [inline samples/texture_lod.html:214]: Should this be `0.0` or `3.0`? Below, when `MAX` is `10.0` and `100.0`, `MIN` is `0.0`.

**Comment by pjcozzi** [comment]
This is a great sample,  but only the top right viewport looks significantly different.  Is it possible to tweak the parameters so each viewport looks different?  There should be enough mip levels.

image

**Comment by shrekshao** [comment]
The other two are originally used to demonstrate bias which is not available here. But sure, I will try to find a way to make the full use of it to make it look differenct

File: samples/texture_lod.html [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+                0.0, 0.0, 0.0, 1.0
+            ]);
+            gl.uniformMatrix4fv(uniformMvpLocation, false, matrix);
+            gl.uniform1i(uniformDiffuseLocation, 0);
+
+            gl.activeTexture(gl.TEXTURE0);
+            
+            gl.bindVertexArray(vertexArray);
+
+            // Bind samplers
```
Comment (trungtle) [inline samples/texture_lod.html]: remove comment

**Comment by shrekshao** [comment]
Ready for a review: 
- Add shader lod bias
- Now the behaviour is fully as expected:
  - top-left: default mipmap
  - top-right: clamped lod
  - bottom: two different lod bias

**Comment by pjcozzi** [comment]
Looks great!  :+1:

**Pull request merged**